pom.xml

```diff
@@ -0,0 +1,734 @@
+        <jackson-databind.version>2.10.5.1</jackson-databind.version>
+
+        <jakarta.annotation-api.version>1.3.5</jakarta.annotation-api.version>
+        <jakarta.activation-api.version>1.2.2</jakarta.activation-api.version>
+        <jakarta.el.version>3.0.4</jakarta.el.version>
+        <jakarta.servlet-api.version>4.0.4</jakarta.servlet-api.version>
+        <jakarta.xml.bind-api.version>2.3.3</jakarta.xml.bind-api.version>
+        <javassist.version>3.28.0-GA</javassist.version>
+        <jaxb-api.version>2.3.1</jaxb-api.version>
+        <jaxws-rt.version>2.3.3</jaxws-rt.version>
+        <jboss-logging.version>3.4.2.Final</jboss-logging.version>
+        <jdbi3.version>3.21.0</jdbi3.version>
+        <jersey.version>2.33</jersey.version>
+        <jetty.version>9.4.43.v20210629</jetty.version>
+        <joda-time.version>2.10.10</joda-time.version>
+        <jsonassert.version>1.5.0</jsonassert.version>
+        <jsch.version>0.1.55</jsch.version>
+        <jsr305.version>3.0.2</jsr305.version>
+        <junit.platform.commons.version>1.8.2</junit.platform.commons.version>
+        <kotlin.version>1.4.10</kotlin.version>
+        <liquibase.version>3.10.3</liquibase.version>
+        <logback.version>1.2.5</logback.version>
+        <logstash.version>6.6</logstash.version>
+        <mongo-java-driver.version>3.12.9</mongo-java-driver.version>
+        <mongo-java-server.version>1.39.0</mongo-java-server.version>
+        <netty.version>4.1.70.Final</netty.version>
+        <okhttp.version>4.9.0</okhttp.version>
+        <postgresql.version>42.2.23</postgresql.version>
+        <servo-core.version>0.13.2</servo-core.version>
+        <spring.version>5.3.13</spring.version>
+        <spring-data-mongodb.version>2.2.13.RELEASE</spring-data-mongodb.version>
+        <stax-ex.version>1.8.3</stax-ex.version>
+        <stax2-api.version>4.2.1</stax2-api.version>
+        <woodstox-core.version>5.1.0</woodstox-core.version>
+        <zookeeper.version>3.7.0</zookeeper.version>
+
+        <!-- Sonar properties -->
+        <sonar.projectKey>kiwiproject_kiwi-bom</sonar.projectKey>
+        <sonar.organization>kiwiproject</sonar.organization>
+        <sonar.host.url>https://sonarcloud.io</sonar.host.url>
+    </properties>
+
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>org.apache.ant</groupId>
+                <artifactId>ant</artifactId>
+                <version>${ant.version}</version>
+            </dependency>
+
+            <dependency>
+                <groupId>org.apache.ant</groupId>
+                <artifactId>ant-launcher</artifactId>
+                <version>${ant.version}</version>
+            </dependency>
+
+            <dependency>
+                <groupId>com.netflix.archaius</groupId>
+                <artifactId>archaius-core</artifactId>
+                <version>${archaius-core.version}</version>
+            </dependency>
+
+            <dependency>
```

Comment:
So for things like `kiwi-test`, `embedded-eureka`, and the test jar for `application-errors` this and the other "testing" libraries need to be here because the parent has them in test scope, but those libraries need them in compile scope

Reply:
I don't think we build a separate test JAR for dropwizard-application-errors. (I think we originally did on the original "rotten tomato" implementation, but I think we decided against that for kiwi.)

Reply:
And ok, this makes sense as to why these are here.